### ISSUE TITLE
Accessibility Issue - Missing Focus style Contrast on Filter Buttons (WCAG 1.4.11 Level AA)

### ISSUE THREAD

**Issue by sherenemorris03** [DFE-Digital/find-information-products-services-frontend#109]
**Description**
**Issue 1 in Accessibility Report**
The focus style on the filter buttons is missing the bold, black underline. This fails to meet minimum colour contrast requirements, making it difficult for some users to perceive the focus state against the white background.
**User Story:**
As a keyboard-only user or someone with low vision,
I want to clearly see which button is focused,
So that I can navigate and interact with the filter options confidently.

**Steps to reproduce**
Navigate to the Products page.
Use the Tab key to move focus to the filter buttons.
Observe the focus style.

**Expected behaviour**
The focused button should have a bold, black underline to meet contrast requirements.

The underline is missing, making the focus style hard to perceive.

**Screenshots**

<img width="484" height="414" alt="Image" src="https://github.com/user-attachments/assets/5f70a2c9-c1f9-4502-84e5-670973d09328" />

<img width="520" height="570" alt="Image" src="https://github.com/user-attachments/assets/65fabcd5-14cf-48d0-99e8-aa0e74299eff" />

**Additional context**
Include the black underline within the focus style to ensure all users can perceive the focus style for the buttons.

Link to issue on Statement Service:
https://accessibility-statements.education.gov.uk/services/DACFEE24-0602-46AD-AE7C-19A23DADA42D/issues/F81139A3-4E18-4EFA-9F0B-20D5AC2C3DDD
